recipes/ntl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "10.3.0" %}
+{% set name = "ntl" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.shoup.net/ntl/ntl-{{ version }}.tar.gz
+  sha256: 717b8d519d66f383a6ae938e255ee4dd103dbab77e4e828dc9cbb0749ec4c1f4
+
+build:
+  number: 0
+  skip: true        # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libtool
+    - automake
+    - gf2x
```

Comment:
Some story about the race condition. Remember to restart this one when `gf2x` is available.